doc/md/fundamentals/1-basic-syntax/5-characters-strings.md

```diff
@@ -0,0 +1,36 @@
+---
+sidebar_position: 5
+---
+
+# Characters & strings
```

Comment:
Maybe it is worth mentioning a conversion between `Text` and a sequence of `Char`s.
I remember someone in the office had a problem with that and got sth like`"[a, b, c]"` instead of `"abc"`